### PR TITLE
Adding a new section including the tools requirements to the docs

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The idea is to set a requirement baseline for tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,19 @@ chat in realtime with the snapd team and our wider community on the
...
 and on [Facebook](https://www.facebook.com/snapcraftio).
 
+## Adding new tools
+
+The tools included in this project are intended to be reused by other projects.
+
+Read the following considerations before adding new tools:
+
+ - Each tool has to be stored in the tools directory
+ - At least 1 spread test needs to be included in the tests directory for each tool
```
Comment (degville) [inline README.md:43]:
```suggestion
 - Each tool needs to be accompanied by at least 1 spread test in `tests/<tool-name>/`
```

File: README.md [hunk 2]
```diff
@@ -33,6 +33,19 @@ chat in realtime with the snapd team and our wider community on the
...
+ - In case the tool is a shell script, this needs to pass the shellcheck
```
Comment (degville) [inline README.md]:
```suggestion
 - If the tool is a shell script, it needs to first pass a [ShellCheck](https://github.com/koalaman/shellcheck) assessment
```

File: README.md [hunk 3]
```diff
@@ -33,6 +33,19 @@ chat in realtime with the snapd team and our wider community on the
...
+ - The tools need to be as generic as possible
```
Comment (degville) [inline README.md]:
```suggestion
 - All tools need to be as generic as possible
```

File: README.md [hunk 4]
```diff
@@ -33,6 +33,19 @@ chat in realtime with the snapd team and our wider community on the
...
+ - A command line interface (CLI) has to be provided by the tool. including a help
```
Comment (degville) [inline README.md]:
```suggestion
- Each tool must also provide a command line interface (CLI), including _help_ output
```

**Review by degville — APPROVED** [review]
This looks good! I've suggested a few tweaks, but there's nothing broken except the typo in the CLI/help output. Also, there's a typo on line 6 outside the scope of this PR: _porpuses_ should be _purposes_.

File: README.md (hunk 1 above)
Comment (sergiocazzolato) [reply README.md:43]: awesome, thanks, all changes applied

**Pull request merged**